ZUGFeRD/InvoiceDescriptor22CIIWriter.cs

```diff
@@ -1266,7 +1266,7 @@ private void _writeOptionalParty(ProfileAwareXmlTextWriter writer, PartyTypes pa
                 case PartyTypes.BuyerTradeParty:
                     break;
                 case PartyTypes.ShipToTradeParty:
-                    if ((this.Descriptor.Profile != Profile.Extended) && (this.Descriptor.Profile != Profile.XRechnung1) && (this.Descriptor.Profile != Profile.XRechnung)) { return; } // extended, XRechnung1, XRechnung profile only
+                    if ((this.Descriptor.Profile != Profile.Extended) && (this.Descriptor.Profile != Profile.Comfort) && (this.Descriptor.Profile != Profile.XRechnung1) && (this.Descriptor.Profile != Profile.XRechnung)) { return; } // extended, XRechnung1, XRechnung profile only
```

Comment:
This is not correct at least not according to ZUGFeRD 2.2:

![grafik](https://github.com/user-attachments/assets/26582033-a821-4ee2-808a-663ff8bb6194)